### PR TITLE
Added documentation on how to create Pull Request with ECA validation

### PR DESCRIPTION
**Pull request by cesarhernandezgt**

### CODE REVIEW

File: docs/how-to-pr.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+= Workflow to make your first Pull Request
+
+Assuming that you already have an Eclipse User account and signed the Eclipse Contribution Agreement, the following is how you can create PR's for Jakarta EE related project:
+
+- If it's the first time you are going to work on the repository: Fork into your account the repository https://github.com/jakartaee/collateral
+
+- Download (you can use the Github UI option `Open With Github Desktop`) your new fork into your computer.
+
+- In your local repository, create a branch for they contribution, example:
```
Comment (kazumura) [inline docs/how-to-pr.adoc:9]: 'they' -> 'the' ?

File: docs/how-to-pr.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+- Add, edit or perform the changes required in your contribution.
+
+- Stage your new files:
+
+   git add .
+
+- Commit your changes locally:
+
+    commit -m "Added Synergy JakartaEE and MicroProfile artwork."
```
Comment (kazumura) [inline docs/how-to-pr.adoc:26]: ==> 
  git commit -m "Added Synergy JakartaEE and MicroProfile artwork."

File: docs/how-to-pr.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+- Assuming your GitHub username is: `CONTRIBUTOR` and the email address used for your Eclipse account is: `CONTRIBUTOR@mail.com`, the following is the line you can use to sign the commit:
+
+    git commit --amend --author="CONTRIBUTOR <CONTRIBUTOR@mail.com>"
+
+- Add bellow the line of commit message removing any trailing blan line:
```
Comment (kazumura) [inline docs/how-to-pr.adoc:32]: 'blan line' -> 'blank lines'

File: docs/how-to-pr.adoc (hunk 2 above)
Comment (ivargrimstad) [inline docs/how-to-pr.adoc:26]: You can do this in one step with the -s switch to the commit

Example:
```
commit -m "Added Synergy JakartaEE and MicroProfile artwork." -s`
```

Before you do so, verify that you have configured Git with the correct email:

```
 git config --get user.email
CONTRIBUTOR@mail.com
```

File: docs/how-to-pr.adoc (hunk 1 above)
Comment (cesarhernandezgt) [reply docs/how-to-pr.adoc]: Fixed in #a618ce9

File: docs/how-to-pr.adoc (hunk 2 above)
Comment (cesarhernandezgt) [reply docs/how-to-pr.adoc]: fixed in #a618ce9

File: docs/how-to-pr.adoc (hunk 3 above)
Comment (cesarhernandezgt) [reply docs/how-to-pr.adoc]: fixed in #a618ce9

File: docs/how-to-pr.adoc (hunk 2 above)
Comment (cesarhernandezgt) [reply docs/how-to-pr.adoc]: Added in #a618ce9

**Comment by cesarhernandezgt** [comment]
Thank you much @kazumura @ivargrimstad for your review. I pushed a new commit with your suggestions.

**Review by kazumura — APPROVED** [review]

**Pull request merged**